### PR TITLE
Add deletion wiki article

### LINKED ISSUE #993: [Doc] Add Delete wiki page
## Description

Add a wiki page under `Concepts -> Decentralized Identifiers (DID)` section describing the deactivation and destruction operations for IOTA DID Documents.

## To-do List

- [ ] Add `Delete` wiki page.

### PR DESCRIPTION
**Pull request by cycraig**
# Description of change
- Add a wiki article on deactivating and destroying IOTA Identities (`Concepts` -> `Decentralized Identifiers (DID)` -> `Delete`).
- Minor fixes to the IOTA DID Method specification.
- Update the `CodeSnippetComponent` to display Node.js code when a Replit is not configured.

## Links to any relevant issues
Fixes #993.

## Type of change

- [x] Documentation

## How the change has been tested
`cd documentation && yarn && yarn start`.

## Change checklist

- [x] I have followed the contribution guidelines for this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: documentation/docs/concepts/decentralized_identifiers/delete.mdx
Comment (eike-hass) [inline documentation/docs/concepts/decentralized_identifiers/delete.mdx:40]: Unnecessary line break between `because`...`it`. Couldn't create a suggestion for it 🙈

File: documentation/docs/concepts/decentralized_identifiers/delete.mdx
Comment (eike-hass) [inline documentation/docs/concepts/decentralized_identifiers/delete.mdx:45]: Unnecessary line break between `reclaiming`...`the`.

File: documentation/docs/concepts/decentralized_identifiers/delete.mdx
Comment (eike-hass) [inline documentation/docs/concepts/decentralized_identifiers/delete.mdx:36]: Unnecessary line break between `update`...`restoring`. Couldn't create a suggestion for it 🙈

File: documentation/docs/concepts/decentralized_identifiers/delete.mdx
Comment (eike-hass) [inline documentation/docs/concepts/decentralized_identifiers/delete.mdx:69]:
```suggestion
Note that historical versions of identities may still be stored off-ledger or on a permanode, so sensitive or
```

File: documentation/docs/specs/did/iota_did_method_spec.md
Comment (eike-hass) [inline documentation/docs/specs/did/iota_did_method_spec.md:202]: unnecessary line break.

File: documentation/docs/specs/did/iota_did_method_spec.md
Comment (eike-hass) [inline documentation/docs/specs/did/iota_did_method_spec.md:215]: unnecessary line break.

**Review by eike-hass — APPROVED** [review]
Good content! Added minor comments regarding formatting, other then that LGTM!

File: documentation/docs/specs/did/iota_did_method_spec.md
Comment (abdulmth) [inline documentation/docs/specs/did/iota_did_method_spec.md:264]:
```suggestion
As with any DID method, identities can be linked if they are used too often and their usage somehow becomes public. See [DID Correlation Risks](https://www.w3.org/TR/did-core/#did-correlation-risks). Additionally, a DID can be correlated with funds if the Alias Output used to store the DID Document or any of its controllers is used for holding, transferring or controlling coins or NFTs. 
```

**Review by abdulmth — APPROVED** [review]
Looks good, thanks for cleaning up the method specs too. One inconsistency is the name of the example which is `delete` while in the wiki it's `destroy`. We might want to change that.

**Comment by cycraig** [comment]
Renaming the example/s will probably have to wait until the resolver and refactors PRs are merged.

**Pull request merged**
